guides/made_for_esphome.rst

```diff
@@ -0,0 +1,52 @@
+.. _made_for_esphome:
+
+Made for ESPHome
+================
+
+.. seo::
+    :description: Information about the "Made for ESPHome" logo
+    :image: /_static/made-for-esphome-black-on-white.png
+
+ESPHome has a great and active community that loves creating and sharing projects.
+You can apply for your project to get the Made for ESPHome stamp of approval.
+This ensures that your project is powered by ESPHome and guarantees a
+minimum level of customizability to users.
+
+When your project matches all requirements of the Made for ESPHome program,
+you can apply for permission to carry the logo by emailing esphome@nabucasa.com
+
+Requirements
+------------
+
+Your product has to match the following requirements:
+ - Your project is powered by ESPHome
+ - Your ESPHome configuration is open source
+ - Users should be able to apply updates if your project sells ready-made devices
+ - Your product name cannot contain **ESPHome** except in the case of *ending with* **for ESPHome**
+
+Logos
+-----
+
+.. figure:: /_static/made-for-esphome-black-on-white.png
+    :align: center
+    :width: 100%
+
+    Made with ESPHome black on white
+
+.. figure:: /_static/made-for-esphome-white-on-black.png
+    :align: center
+    :width: 100%
+
+    Made with ESPHome white on black
+
+.. figure:: /_static/made-for-esphome-black-on-transparent.png
+    :align: center
+    :width: 100%
+
+    Made with ESPHome black on transparent
+
+.. figure:: /_static/made-for-esphome-white-on-transparent.png
+    :align: center
+    :width: 100%
+
+    Made with ESPHome white on transparent
```

Comment:
```suggestion
.. raw:: html

    <img src="/_static/made-for-esphome-white-on-transparent.png" style="background: repeating-conic-gradient(#999 0% 25%, transparent 10% 50%) 50% / 10px 10px;">
    <center>Made with ESPHome white on transparent</center>
```

An alternative that will give:
![Screenshot from 2021-09-07 11-10-43](https://user-images.githubusercontent.com/15093472/132318227-6155fb6d-6f99-4d8f-9156-7698d071f886.png)